lang/languages.xml

```diff
@@ -17,42 +17,42 @@ Deutsch		(German)
+        <language display_name="日本語" locale="ja_JP" wallet_language="日本語" flag="/lang/flags/jp.png" qs="none"/>
+        <language display_name="Nederlands" locale="nl_NL" wallet_language="Nederlands" flag="/lang/flags/nl.png" qs="none"/>
+        <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/pl.png" qs="none"/>
+        <!-- <language display_name="Polski" locale="pl_PL" wallet_language="English" flag="/lang/flags/pl.png" qs="none"/> -->
+        <language display_name="Português (PT)" locale="pt-pt_PT" wallet_language="Português" flag="/lang/flags/pt.png" qs="none"/>
+        <language display_name="Português (BR)" locale="pt-br_BR" wallet_language="Português" flag="/lang/flags/br.png" qs="none"/>
+        <language display_name="Русский язык" locale="ru_RU" wallet_language="русский язык" flag="/lang/flags/ru.png" qs="none"/>
+        <language display_name="简体中文 (中国)" locale="zh-cn_CN" wallet_language="简体中文 (中国)" flag="/lang/flags/cn.png" qs="none"/>
+        <language display_name="繁體中文 (台灣)" locale="zh-tw_CN" wallet_language="English" flag="/lang/flags/tw.png" qs="none"/>
+        <language display_name="עברית" locale="he_HE" wallet_language="English" flag="/lang/flags/il.png" qs="none"/>
+        <language display_name="한국어" locale="ko_KO" wallet_language="English" flag="/lang/flags/kr.png" qs="none"/>
+        <language display_name="Română" locale="ro_RO" wallet_language="English" flag="/lang/flags/ro.png" qs="none"/>
+        <language display_name="Dansk" locale="da_DK" wallet-language="English" flag="/lang/flags/dk.png" qs="none"/>
+        <language display_name="Česky" locale="cs_CZ" wallet_language="English" flag="/lang/flags/cz.png" qs="none"/>
+        <language display_name="Slovensky" locale="sk_SK" wallet_language="English" flag="/lang/flags/sk.png" qs="none"/>
+        <language display_name="العربية" locale="ar_AR" wallet_language="English" flag="/lang/flags/eg.png" qs="none"/>
+        <language display_name="Slovenski" locale="sl_SI" wallet_language="English" flag="/lang/flags/si.png" qs="none"/>
+        <language display_name="Srpski" locale="rs_RS" wallet_language="English" flag="/lang/flags/rs.png" qs="none"/>
         <language display_name="Català" locale="cat_ES" wallet_language="English" flag="/lang/flags/catalonia.png" qs="none"/>
         <language display_name="Türkçe" locale="tr_TR" wallet_language="English" flag="/lang/flags/tr.png" qs="none"/>
-        <language display_name="Українська" locale="uk_UA" wallet_language="English" flag="/lang/flags/ukraine.png" qs="none"/>
+        <language display_name="Українська" locale="uk_UA" wallet_language="English" flag="/lang/flags/ua.png" qs="none"/>
         <language display_name="Lietuvių" locale="lt_LT" wallet_language="English" flag="/lang/flags/lt.png" qs="none"/>
         <language display_name="Suomi" locale="fi_FI" wallet_language="English" flag="/lang/flags/finland.png" qs="none"/>
```

Comment:
This line should be adapted as well ot this flag won't display properly: `flag="/lang/flags/fi.png"`